### ISSUE TITLE
loss function in wrappers.py

### ISSUE THREAD

**Issue by yimjiyeon** [gene-chou/EFPN-detectron2#3]
Which loss function should I use?

Can I use Ce loss?

def cross_entropy(input, target, *, reduction="mean", **kwargs):
    """
    Same as `torch.nn.functional.cross_entropy`, but returns 0 (instead of nan)
    for empty inputs.
    """
    if target.numel() == 0 and reduction == "mean":
        return input.sum() * 0.0  # connect the gradient
    return F.cross_entropy(input, target, **kwargs)
